libs/SmartStore/SmartStore/Classes/SFSDKStoreConfig.m

```diff
@@ -44,7 +44,7 @@ @implementation SFSDKStoreConfig
 - (nullable id)initWithResourceAtPath:(NSString *)path {
     self = [super init];
     if (self) {
-        NSDictionary *config = [SFSDKResourceUtils loadConfigFromFile:path];
+        NSDictionary *config = [SFSDKResourceUtils loadConfigFromFile:path error:nil];
         self.soupsConfig = config == nil ? nil : config[kStoreConfigSoups];
```

Comment:
Code does not always expect a config - so if no config found, we don't need to log any error.